pkg/strategy/factorzoo/correlation.go

```diff
@@ -0,0 +1,103 @@
+package factorzoo
+
+import (
+	"fmt"
+	"github.com/c9s/bbgo/pkg/indicator"
+	"github.com/c9s/bbgo/pkg/types"
```

Comment:
Maybe split sections by sources?
Built-in
External
Internal

Reply:
sure

Reply:
however my Goland IDE forced me to sort like above code.

`tips: Imports are not sorted`

Reply:
that's weired